src/main/resources/static/overview.js

```diff
@@ -268,6 +275,9 @@ function getToolTip(rating) {
         case 'D':
             fix = 'high';
             break;
+        case 'E':
```

Comment:
If you do end up changing the rating for critical, make sure to change it here as well.